tests/chart_tests/test_authsidecar.py

```diff
@@ -156,7 +164,11 @@ def test_authSidecar_houston_configmap_with_annotation(self, kube_version):
             kube_version=kube_version,
             values={
                 "global": {
-                    "authSidecar": {"enabled": True},
+                    "authSidecar": {
+                        "enabled": True,
+                        "repository": "nginxinc/nginx-unprivileged",
```

Comment:
If we're going to set this then there should be an assertion against it.

Reply:
okay will make those changes